x/concentrated-liquidity/math/tick_test.go

```diff
@@ -188,70 +226,45 @@ func (suite *ConcentratedMathTestSuite) TestPriceToTick() {
 		tickExpected       string
 		expectedError      error
 	}{
-		"50000 to tick with -4 k at price one": {
-			price:              sdk.NewDec(50000),
-			exponentAtPriceOne: sdk.NewInt(-4),
-			tickExpected:       "400000",
-		},
-		"5.01 to tick with -2 k at price one": {
-			price:              sdk.MustNewDecFromStr("5.01"),
-			exponentAtPriceOne: sdk.NewInt(-2),
-			tickExpected:       "401",
-		},
-		"50000.01 to tick with -6 k at price one": {
-			price:              sdk.MustNewDecFromStr("50000.01"),
+		"BTC <> USD at exponent at price one of -6, tick spacing 100, tick 38035200 -> price 30352": {
 			exponentAtPriceOne: sdk.NewInt(-6),
-			tickExpected:       "40000001",
-		},
-		"0.090000889 to tick with -8 k at price one": {
-			price:              sdk.MustNewDecFromStr("0.090000889"),
-			exponentAtPriceOne: sdk.NewInt(-8),
-			tickExpected:       "-999991110",
+			price:              sdk.MustNewDecFromStr("30352"),
+			tickExpected:       "38035200",
 		},
-		"0.9998 to tick with -4 k at price one": {
-			price:              sdk.MustNewDecFromStr("0.9998"),
-			exponentAtPriceOne: sdk.NewInt(-4),
-			tickExpected:       "-20",
-		},
-		"53030.10 to tick with -5 k at price one": {
-			price:              sdk.MustNewDecFromStr("53030.1"),
-			exponentAtPriceOne: sdk.NewInt(-5),
-			tickExpected:       "4030301",
-		},
-		"max spot price and minimum exponentAtPriceOne": {
-			price:              types.MaxSpotPrice,
-			exponentAtPriceOne: sdk.NewInt(-1),
-			tickExpected:       "3420",
+		"BTC <> USD at exponent at price one of -6, tick spacing 100, tick 38035300 + 100 -> price 30353": {
+			exponentAtPriceOne: sdk.NewInt(-6),
+			price:              sdk.MustNewDecFromStr("30352"),
+			tickExpected:       "38035200",
```

Comment:
These need to change to what the test case says

Reply:
addressed